liblangutil/SourceReferenceFormatterHuman.cpp

```diff
@@ -73,7 +73,7 @@ void SourceReferenceFormatterHuman::printSourceLocation(SourceReference const& _
 
 	// line 0: source name
 	frameColored() << string(leftpad, ' ') << "--> ";
-	m_stream << _ref.sourceName << ":" << (_ref.position.line + 1) << ":" << (_ref.position.column + 1) << ": " << '\n';
+	m_stream << _ref.sourceName << ":" << (_ref.position.line + 1) << ":" << (_ref.position.column + 1) << ":" << '\n';
```

Comment:
Does it make sense to add a test case expects warning/error without trailing spaces?

Reply:
I'd added/adjusted a test case but the whole feature doesn't have a single test and I felt this little change didn't warrant writing one now.